src/pages/dashboard/analysis/utils/Yuan.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { yuan } from '../components/Charts';
+/**
+ * 减少使用 dangerouslySetInnerHTML
+ */
+export default class Yuan extends React.Component<{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)